src/util.cxx

```diff
@@ -9,14 +9,139 @@
 
 #include "util.h"
 
+#include <vector>
+#include <cctype>
+#include <cmath>
+
 NAMESPACE__THITTAM_UTIL__START
 
+
+std::string getString(const std::string &s, unsigned &pos) {
```

Comment:
1. We use 'snake_case' for function and variable names. So, please keep with that consistently.
2. See how we place curly braces in the start of the function, in other code. Please maintain the same for consistency.